### PR TITLE
fixed riak_control test to handle riak_ts

### PR DESCRIPTION
**Pull request by korry8911**
@javajolt ok the diff looks better now, fixed riak_control to handle riak_ts version the same way it handles riak-2. and riak_ee-2.
Test results:
Test Results:
## riak_control-eleveldb: pass

0 Tests Failed
1 Tests Passed
That's 100.0% for those keeping score

### CODE REVIEW

File: tests/riak_control.erl [hunk 1]
```diff
@@ -114,6 +114,7 @@ verify_resource(Node0, Resource) ->
     Node = rt:http_url(Node0),
     Output = os:cmd(io_lib:format("curl -s -S ~s~p", [Node, Resource])),
     lager:info("Verifying node ~p resource ~p.", [Node, Resource]),
+    lager:info("~p~n", [mochijson2:decode(Output)]),
```
Comment (hazen) [inline tests/riak_control.erl]: Not sure we want to leave in this debug output.  If we do, let's label it like in the above line.

**Review by hazen — CHANGES_REQUESTED** [review]
Functionality looks good, but just has the extra output we might not want.

**Comment by korry8911** [comment]
@javajolt forgot that was in there, it has been removed

**Review by hazen — APPROVED** [review]

**Comment by hazen** [comment]
:+1:

**Pull request merged**